beacon_node/beacon_chain/src/beacon_chain.rs

```diff
@@ -1037,14 +1037,20 @@ impl<T: BeaconChainTypes> BeaconChain<T> {
                 );
             }
 
-            return Err(Error::InconsistentPayloadReconstructed {
-                slot: blinded_block.slot(),
-                exec_block_hash,
-                canonical_payload_root: execution_payload_header.tree_hash_root(),
-                reconstructed_payload_root: header_from_payload.tree_hash_root(),
-                canonical_transactions_root: execution_payload_header.transactions_root(),
-                reconstructed_transactions_root: header_from_payload.transactions_root(),
-            });
+            if execution_payload_header.transactions_root()
+                != header_from_payload.transactions_root()
+            {
+                //FIXME(sean) we're not decoding blobs txs correctly yet
+            } else {
+                return Err(Error::InconsistentPayloadReconstructed {
```

Comment:
This will return the error for pre 4844 blocks where payload is reconstructed correctly right?

Reply:
Ahhhh whoops nice catch! I'll simplify the logic by just skipping this veriifcation entirely if we are in 4844